src/index.js

```diff
@@ -0,0 +1,42 @@
+const inquirer = require("inquirer");
+const utilities = require("./utilities");
+const writeReadme = require("./readme");
+const { questions, installQuestions, testQuestions } = require("./questions");
+
+const start = async () => {
+  const readmeAnswers = await inquirer.prompt(questions);
```

Comment:
You could destructure the `installationIncluded` and `testIncluded` keys, using the rest operator to spread the rest of the answers at the end: 

`const { installationIncluded , testIncluded , ...otherAnswers } = await inquirer.prompt(questions)`